### PR TITLE
ODC-7549,ODC-7550,ODC-7551,ODC-7552: Add Triggers lists

### PR DESCRIPTION
**Pull request by vikram-raj**

### CODE REVIEW

File: console-extensions.json [hunk 1]
```diff
@@ -345,5 +345,88 @@
...
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "pipelinetriggers",
+      "perspective": "admin",
+      "section": "pipelines",
+      "name": "%Triggers%",
+      "href": "/triggers",
```
Comment (lokanandaprabhu) [inline console-extensions.json:425]: Can you add insertBefore ?

"insertAfter": "pipeline-tasks",

File: src/components/triggers-list/EventListenersRow.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  const onToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = () => {
+    setIsOpen(false);
+  };
+
+  const dropdownItems = K8sCommonKebabMenu(obj, model);
+
```
Comment (lokanandaprabhu) [inline src/components/triggers-list/EventListenersRow.tsx:41]: Screenshot 2024-03-25 at 12 43 00 PM

In TriggerTemplate list page edit and delete button in kebab menu showing TriggerBinding kind

**Comment by lokanandaprabhu** [comment]
Screenshot 2024-03-25 at 12 52 15 PM
Screenshot 2024-03-25 at 12 52 39 PM


Design is different compare to existing design. Filters is not part of search resource list page in existing design and v1beta1 version is not added for TiggerTemplates

File: console-extensions.json [hunk 3]
```diff
@@ -345,5 +345,88 @@
...
+      "required": ["HIDE_STATIC_PIPELINE_PLUGIN_TRIGGERBINDINGS_LIST"]
+    }
+  },
+  {
+    "type": "console.page/resource/list",
+    "properties": {
+      "model": {
+        "group": "triggers.tekton.dev",
+        "version": "v1alpha1",
+        "kind": "TriggerTemplate"
```
Comment (lokanandaprabhu) [inline console-extensions.json:383]: Screenshot 2024-03-25 at 12 52 15 PM

"version": "v1beta1" version is not added or need to replace v1alpha1 to v1beta1 ?

**Comment by vikram-raj** [comment]
@lokanandaprabhu  Addresses the review comments PTAL again.

**Review by lokanandaprabhu — APPROVED** [review]

**Pull request merged**